### PR TITLE
API Tests

### PR DESCRIPTION
**Pull request by sdevalapurkar**
# Overview

API Tests

### CODE REVIEW

**Review by NickPhura — DISMISSED** [review]

**Comment by rstens** [comment]
Is there a reason why this is not merged yet?

**Comment by sdevalapurkar** [comment]
> Is there a reason why this is not merged yet?

was waiting on review/approval
didnt wanna bug the team so just turned it into draft and continuing to work on it and add more tests

**Review by anissa-agahchen — APPROVED** [review]

**Pull request merged**
